File: GTTS/GTTS.Web/ClientApp/src/app/models/INL.TrainingService.Models/get-uspartner-agencies-at-business-unit_item.ts


import { IGetUSPartnerAgenciesAtBusinessUnit_Item } from './iget-uspartner-agencies-at-business-unit_item';

export class GetUSPartnerAgenciesAtBusinessUnit_Item implements IGetUSPartnerAgenciesAtBusinessUnit_Item {
  
	public AgencyID: number = 0;
	public Name: string = "";
	public BusinessUnitID: number = 0;
	public Acronym: string = "";
	public BusinessUnitName: string = "";
	public BusinessUnitActive: boolean = false;
	public USPartnerAgenciesBusinessUnitActive: boolean = false;
  
}

File: GTTS/INL.TrainingService.Models/GetUSPartnerAgenciesAtBusinessUnit_Item.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace INL.TrainingService.Models
{
    public class GetUSPartnerAgenciesAtBusinessUnit_Item : IGetUSPartnerAgenciesAtBusinessUnit_Item
    {
        public int AgencyID { get; set; }
        public string Name { get; set; }
        public long BusinessUnitID { get; set; }
        public string Acronym { get; set; }
        public string BusinessUnitName { get; set; }
        public bool BusinessUnitActive { get; set; }
        public bool USPartnerAgenciesBusinessUnitActive { get; set; }
    }
}
